devices/Bmxx80/Bme280.cs

```diff
@@ -141,10 +141,10 @@ private RelativeHumidity CompensateHumidity(int adcHumidity)
         {
             // The humidity is calculated using the compensation formula in the BME280 datasheet.
             double varH = TemperatureFine - 76800.0;
-            varH = (adcHumidity - (_bme280Calibration.DigH4 * 64.0 + _bme280Calibration.DigH5 / 16384.0 * varH)) *
-                (_bme280Calibration.DigH2 / 65536.0 * (1.0 + _bme280Calibration.DigH6 / 67108864.0 * varH *
-                                               (1.0 + _bme280Calibration.DigH3 / 67108864.0 * varH)));
-            varH *= 1.0 - _bme280Calibration.DigH1 * varH / 524288.0;
+            varH = (adcHumidity - ((_bme280Calibration.DigH4 * 64.0) + ((_bme280Calibration.DigH5 / 16384.0) * varH))) *
```

Comment:
@Ellerbach 
SA1407 example here.

Reply:
ok, case by case to release the rule. Now, that said, it's a rule that brings clarity and avoid mistakes. But I get your point. So make it the best with flexibility.

Reply:
Sorry, but I think i didn't get you. Should I remove the rule or not?